### PR TITLE
add support for returning null in infoTag

### PR DESCRIPTION
**Pull request by raphaelblum**
before (when returning null):
image (2)

after (when returning null):
image

### CODE REVIEW

File: packages/admin/cms-admin/src/documents/types.ts [hunk 1]
```diff
@@ -44,5 +44,5 @@ export interface DocumentInterface<
     inputToOutput?: (input: DocumentInput, context: { idsMap: IdsMap }) => DocumentOutput;
     menuIcon: (props: SvgIconProps<"svg">) => JSX.Element;
     hideInMenuIcon?: (props: SvgIconProps<"svg">) => JSX.Element;
-    InfoTag?: React.ComponentType<{ page: PageTreePage }>;
+    InfoTag?: ({ page }: { page: PageTreePage }) => React.ReactNode;
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/documents/types.ts:47]:
```suggestion
    InfoTag?: React.FunctionComponent<{ page: PageTreePage }>;
```

File: packages/admin/cms-admin/src/pages/pageTree/PageLabel.tsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ interface PageLabelProps {
 const PageLabel: React.FunctionComponent<PageLabelProps> = ({ page, disabled, onClick }) => {
     const { documentTypes } = usePageTreeContext();
     const documentType = documentTypes[page.documentType];
+    const infoTag = documentType.InfoTag && documentType.InfoTag({ page });
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/pages/pageTree/PageLabel.tsx:20]:
```suggestion
    const infoTag = documentType.InfoTag?.({ page });
```

**Review by johnnyomair — COMMENTED** [review]
It should still be possible to use Hooks in the info tag, for instance `useQuery` or `useState`. Could you please verify that this is still working?

**Comment by johnnyomair** [comment]
As discussed: Remove the default chip from the PageLabel component, as the InfoTag component should render it itself (PageLabel still creates the margin-left). Target `next` branch. Please provide an example in the changelog on how to update

**Comment by raphaelblum** [comment]
@johnnyomair this PR can be reviewed again

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**